src/Hledger/Flow/Reports.hs

```diff
@@ -59,7 +59,7 @@ generateReports' opts ch = do
   ownerParams <- ownerParameters opts ch owners
   let ownerWithAggregateParams = (if length owners > 1 then [aggregateParams] else []) ++ ownerParams
   let sharedOptions = ["--pretty-tables", "--depth", "2"]
-  let ownerWithAggregateReports = List.concat $ fmap (reportActions opts ch [incomeStatement sharedOptions, balanceSheet sharedOptions]) ownerWithAggregateParams
+  let ownerWithAggregateReports = List.concat $ fmap (reportActions opts ch [incomeStatement incomeMonthlyStatement sharedOptions, balanceSheet sharedOptions]) ownerWithAggregateParams
```

Comment:
The list `[incomeStatement sharedOptions, balanceSheet sharedOptions]` (baseline code) is a list of `ReportGenerator`s

In Haskell `incomeStatement sharedOptions` means we are supplying one argument (`sharedOptions`) to the function named `incomeStatement`, and in this case the result type is then `ReportGenerator`.

To use your new monthly function, I think you should add an element to the list like this:
`[incomeStatement sharedOptions, incomeMonthlyStatement sharedOptions, balanceSheet sharedOptions]`

Reply:
ah thanks, I overlooked that yesterday. This also explains where the sharedOptions come from. I was wondering where that variable comes from.